.github/workflows/build.yml

```diff
@@ -107,3 +103,36 @@ jobs:
 
       - name: Build package
```

Comment:
Remove build step from `test_full` job

Reply:
Also, will there be any issues with uploading the coverage report twice?

Reply:
Also, should test on windows on the master branch aswell. So you should make the same changes in the `release` workflow.

Reply:
I think the coverage report will just overwrite the last one.

Reply:
Updated the release workflow now